network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_block.html

```diff
@@ -5,15 +5,15 @@
 <div class="col-lg-6 col-12 mb-5">
   <div class="person-card">
     <div class="thumbnail-wrapper">
-      <a href="https://www.mozillapulse.org/profile/{{ profile.profile_id }}" class="d-block headshot-container">
+      <a href="https://www.mozillapulse.org/profile/{{ profile.profile_id|stringformat:"s" }}" class="d-block headshot-container">
```

Comment:
I've filed https://github.com/mozilla/foundation.mozilla.org/issues/7355 to switch this behaviour around, because value templating should not perform transforms unless told to. This codebase already has too much Django magic ;)